appcues/src/main/java/com/appcues/ui/primitive/OptionSelectPrimitive.kt

```diff
@@ -127,28 +127,10 @@ private fun List<OptionSelectPrimitive.OptionItem>.ComposeSelections(
     selectedColor: Color?,
     unselectedColor: Color?,
     accentColor: Color?,
-    valueSelectionChanged: (Set<String>) -> Unit,
+    itemSelected: (String) -> Unit,
```

Comment:
some simplifications here, as we're just passing each new selected item as it occurs - the form state now manages the logic around adding/removing from the set, based on the rules defined for the control.